spk/links/Makefile

```diff
@@ -9,7 +9,7 @@ MAINTAINER = SynoCommunity
 DESCRIPTION = Links is a web browser running in text mode.
 DISPLAY_NAME = Links
 STARTABLE = no
-BETA = 1
+CHANGELOG = "1. Update links to 2.21.<br/>2. Update openssl to 1.1."
```

Comment:
I think it would make sense to move this tool in synocli-net

Reply:
@ymartin59 agree, this should be done in #4195, I updated #4211 accordingly.